src/encoded/search/lucene_builder.py

```diff
@@ -565,6 +404,137 @@ def initialize_field_filters(request, principals, doc_types):
+                MUST_NOT: []
+            }
+        }
+
+        # Maps a nested path to a 2-tuple of it's key (must/must_not) and index
+        nested_path_to_index_map = {}
+
+        # Build array of key, (field, query) so we can process the filters in a single pass
+        filters_to_work_on = []
+        if must_nested_filters:
+            filters_to_work_on += zip([MUST] * len(must_nested_filters), must_nested_filters)
+        if must_not_nested_filters:
+            filters_to_work_on += zip([MUST_NOT] * len(must_not_nested_filters), must_not_nested_filters)
+
+        # Process key (must/must_not), field (target of search), query (condition)
+        # iteratively building nested_query
+        for key, (field, query) in filters_to_work_on:
+            nested_path = find_nested_path(field, es_mapping)
+
+            # if we've never seen this path before, bootstrap a sub-query for it
+            if nested_path not in nested_path_to_index_map:
+
+                # set in tracking
+                new_index = len(nested_query[BOOL][key])
+                nested_path_to_index_map[nested_path] = (key, new_index)
+
+                # if the query has multiple parts, they must be appropriately combined
+                if len(query) > 1:
+                    combined_query = {
+                        BOOL: {
+                            key: []
+                        }
+                    }
+                    for sub_query in query:
+                        if BOOL not in sub_query:
+                            combined_query[BOOL][key].append(sub_query)
+
+                        # sub_query has duplicated bool -> must, so traverse down to examine
+                        # the actual sub-query and propagate it
+                        else:
+                            # can only contain one sub_query but does have list structure
+                            try:
+                                _sub_query = sub_query[BOOL][MUST]
+                            except KeyError:
+                                _sub_query = sub_query[BOOL][MUST_NOT]
+                            if isinstance(_sub_query, list):
+                                if len(_sub_query) > 1:
+                                    raise QueryConstructionException(
+                                        query_type='nested',
+                                        func='handle_nested_filters_v2',
+                                        msg='Unexpected query structure: %s' % _sub_query
+                                    )
+                                _sub_query = _sub_query[0]
+                            combined_query[BOOL][key].append(_sub_query)
+                    nested_query[BOOL][key].append(cls.build_nested_query(nested_path, combined_query))
+
+                # if a single sub-query, can be added directly
+                else:
+                    nested_query[BOOL][key].append(cls.build_nested_query(nested_path, query))
```

Comment:
This comment is out of order and follows on the comments below, but I believe this is where the trouble is being introduced with respect to the nested query being built as a list when starting with an `exists` type query. Consider checking to see if the `query` is a list as you do in lines 482-498 above for `_sub_query`. You may have to introduce a similar check for lists at line 472, though I'm less confident of that. 
```suggestion
                    if isinstance(query, list):
                        query = query[0]
                    nested_query[BOOL][key].append(cls.build_nested_query(nested_path, query))
```